assets/e2e/cypress/utils/graphql-test-utils.ts

```diff
@@ -0,0 +1,23 @@
+import { CyHttpMessages } from 'cypress/types/net-stubbing';
+
+// Utility to match GraphQL mutation based on the operation name
+export const hasOperationName = (req: CyHttpMessages.IncomingHttpRequest, operationName: string) => {
+  const { body } = req
+  return (
+    body.hasOwnProperty('operationName') && body.operationName === operationName
```

Comment:
do we need to use `hasOwnProperty` here?

Reply:
Yeah that is what allows us to intercept specific POST requests and use them in `cy.wait(@gqlCreateBuildMutation)` for example.
This code is straight from the Cypress docs for using it with GraphQL.

Reply:
oh I just assumed `body.operationName === operationName` would still be falsy if there's no operationName property